examples/iam-role-for-service-accounts-eks/main.tf

```diff
@@ -136,6 +136,22 @@ module "efs_csi_irsa_role" {
   tags = local.tags
 }
 
+module "s3_csi_irsa_role" {
```

Comment:
lets change all of the `s3_csi` references to `mountpoint_s3_csi`

Reply:
changed :)